### PR TITLE
Fixed creator for website

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| BC breaks        | none
| Documentation PR | none

__Related:__

* https://github.com/sulu-io/sulu-document-manager/pull/25

### CODE REVIEW

File: src/Sulu/Component/PHPCR/PathCleanup.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ class PathCleanup implements PathCleanupInterface
      */
     public function cleanup($dirty, $languageCode)
     {
-        $clean = strtolower($dirty);
+        $clean = mb_strtolower($dirty);
```
Comment (wachterjohannes) [inline src/Sulu/Component/PHPCR/PathCleanup.php]: fixes local failing testcases ... http://php.net/manual/de/function.mb-strtolower.php#refsect1-function.mb-strtolower-unicode

**Comment by wachterjohannes** [comment]
@danrot this should maybe released as HOTFIX ...

File: src/Sulu/Component/PHPCR/PathCleanup.php (hunk 1 above)
Comment (danrot) [inline src/Sulu/Component/PHPCR/PathCleanup.php]: I don't know, the manual also says that this is a very slow function... Maybe we should try to get the locale settings right on our machines?

Comment (wachterjohannes) [inline src/Sulu/Component/PHPCR/PathCleanup.php]: hmm ok ... but at least its a "starting" point (: i will reset this commit!

**Pull request merged**